pages/about.js

```diff
@@ -37,9 +37,9 @@ const AboutPage = ({ lang }) => (
         <div className="about-page__description-text about-page__team">
           {team.map(({ image, name, role }) => (
             <div key={name} className="about-page__teammate teammate">
-              <img className="teammate__pic" src={image} alt={localize(name, lang)} />
-              <div className="teammate__name">{localize(name, lang)}</div>
-              <div className="teammate__role">{localize(role, lang)}</div>
+              <img className="teammate__pic" src={image} alt={name[lang || 'be']} />
+              <div className="teammate__name">{name[lang || 'be']}</div>
+              <div className="teammate__role">{role[lang || 'be']}</div>
```

Comment:
why no `localize`?

Reply:
`localize` is retrieving data from another table. for the team we don't use i18n